packages/locales/src/locales/en.json

```diff
@@ -208,5 +208,8 @@
   "unquoted-string": "an unquoted string",
   "unsorted-keys": "Unsorted keys",
   "uuid": "a UUID",
-  "vector": "a vector"
+  "vector": "a vector",
+  "parser.macro.key": "a macro key",
+  "parser.macro.at-least-one": "at least one macro",
```

Comment:
Shouldn't it be "at least one macro argument"?

"Macro" is more the whole file, or maybe it could be argued the line is a macro, referring to a macro argument as just "a macro" seems strange to me, at least I have never heard it used that way.